packages/2d/src/scenes/Scene2D.ts

```diff
@@ -117,6 +117,22 @@ export class Scene2D extends GeneratorScene<View2D> implements Inspectable {
     }
   }
 
+  public transformMousePosition(
+    x: number,
+    y: number,
+    size?: {width: number; height: number},
+  ): Vector2 | null {
+    if (size !== undefined) {
```

Comment:
nit: `if (size)`

Reply:
This code section was removed. Resolving comment.